.gitlab-ci.yml

```diff
@@ -80,6 +80,24 @@ build_wheel_linux_python36:
     paths:
       - target/
 
+build_wheel_linux_python37:
+  tags:
+    - docker
+  services:
+    - docker:18.09-dind
+  image: docker:18.09
+  stage: build
+  variables:
+    DOCKER_HOST: tcp://docker:2375/
+    DOCKER_DRIVER: overlay2
+  script:
+    - apk add bash
+    - bash scripts/make_wheel.sh --skip_test --skip_cpp_test --build_number=$CI_PIPELINE_ID --num_procs=4 --docker-python3.7 --skip_doc
```

Comment:
Do we want to add the `-e` option, like in #2625?

Reply:
Good call